database_connection_setup.rb

```diff
@@ -1,11 +1,10 @@
 require './lib/database_connection'
 
-def setup_database
+
```

Comment:
dont we want this to be a method?

Reply:
For us, it wasn't working because it was defined as a private method - we realised we only needed it as a simple if else statement to determine which database to connect to!

Reply:
okiiii ❤️